model/generate.go

```diff
@@ -16,6 +16,7 @@ const (
 	maxGeneratedBuildVariants = 100
 	maxGeneratedTasks         = 1000
 	GenerateTasksCommandName  = "generate.tasks"
+	CreateHostCommandName     = "createhost"
```

Comment:
should we put these in globals? and have a global slice for the group?

Reply:
done